### DISCUSSION TITLE
How can I use the Duckdb to generate queries to implement the TPC-DS

### DISCUSSION THREAD

**Discussion by woshimajintao** [duckdb/duckdb#9410] [Q&A]
How can I use the Duckdb to generate queries to implement the TPC-DS just like this one?
https://github.com/duckdb/duckdb/tree/main/extension/tpcds/dsdgen/queries
I am not skilled at the command line.I need someone's help.
 I would like to know how can you generete the queries?

**Comment by woshimajintao** [comment]
I had the solution to it.

**Comment by l1t1** [comment]
what is your solution?
